chapters.md

```diff
@@ -18,26 +18,20 @@ As an example, consider a `Parent ChapterAtom Element` that has its `ChapterFlag
 Three `Edition Flags` are defined to describe the behavior of the `EditionEntry Element`: `EditionFlagHidden`, `EditionFlagDefault` and `EditionFlagOrdered`.
 
 
-#### `EditionFlagHidden`
+#### EditionFlagHidden
 
 The `EditionFlagHidden Flag` behaves similarly to the `ChapterFlagHidden Flag`: if `EditionFlagHidden` is set to `true`, its `Child ChapterAtoms Elements` MUST also be interpreted as if their `ChapterFlagHidden` is also set to `true`, regardless of their own `ChapterFlagHidden Flags`. If `EditionFlagHidden` is toggled by a `Control Track` to `false`, the `ChapterFlagHidden Flags` of the `Child ChapterAtoms Elements` SHALL determine whether the `ChapterAtom` is hidden or not.
 
 
-#### `EditionFlagDefault`
+#### EditionFlagDefault
 
-It is recommended that only one Edition's `EditionFlagDefault Flag` is set to `true`. If more then one `Default Edition` is present, the first `Default Edition` MUST be used.
-If an `EditionFlagDefault Flag` is set to `true`, this `Edition` MUST be used. When all `EditionFlagDefault Flags` are set to `false`, the first Edition MUST be used.
+It is RECOMMENDED that no more than one `Edition` have an `EditionFlagDefault Flag` set to `true`. The first `Edition` with the `EditionFlagDefault Flag` set to `true` is the `Default Edition`. When all `EditionFlagDefault Flags` are set to `false`, then the first `Edition` is the `Default Edition`.
 
+#### EditionFlagOrdered
 
-#### `EditionFlagOrdered`
+The `EditionFlagOrdered Flag` is a significant feature as it enables an Edition of `Ordered Chapters` which define and arrange a virtual timeline rather than simply labeling points within the timeline. For example, with `Editions` of `Ordered Chapters` a single `Matroska file` can present multiple edits of a film without duplicating content. Alternatively if a videotape is digitized in full, one `Ordered Edition` could present the full content (including colorbars, countdown, slate, a feature presentation, and black frames), while another `Edition` of `Ordered Chapters` can use `Chapters` that only mark the intended presentation with the colorbars and other ancilary visual information excluded. If an `Edition` of `Ordered Chapters` is enabled then the Matroska Player MUST play those Chapters in their stored order from the timecode marked in the `ChapterTimeStart Element` to the timecode marked in to `ChapterTimeEnd Element`.
```

Comment:
Edition is capitalized but not backticked in the first line/sentence

Reply:
I'd like more clarity that the define/arrange description in the first sentence is speaking of the `Ordered Chapters` in general and not the `Edition` -- otherwise should be defines/arranges

Reply:
And Chapters in the last sentence is capitalized but not backticked as well!

Reply:
I think from the schema it feels like the concept should be called `Ordered Edition` but `Ordered Chapters` is what has more common usage. will backtick later